app/services/import/csv/activity_actual_refund_comment/row_service.rb

```diff
@@ -15,10 +15,10 @@ def import!
 
       return false if @activity.nil?
 
-      return false if @row.empty?
-
       return false unless authorise_activity(@activity)
 
+      return Import::Csv::ActivityActualRefundComment::SkippedRow.new(@row) if @row.empty?
```

Comment:
Would it make sense to use the SkippedRow for unrecognised rows? (Where `type_of_row` is not one of the three expected types.)

Are unrecognised rows surfaced to the user in any way?

Reply:
We definately could, currently the guard on line 20 means the `create_record` method will not get called if the row considers itself empty and so skipped, this would mean refacotring and renaming - any suggestions?

No, the skipped rows are not surfaced at this point, nothing is because this class is not actuall in use yet, we'll be looking to update the UI later once we and DSIT are happy with the new import and the changes to expectation on users - the loose plan right now is to surface all the types of rows in seperate tables upon successful import, so:

- actuals
- refunds
- activity comments
- skipped rows

But this all come later.